03_controls/03_pairing/05_pairing_no_interaction.en.md

```diff
@@ -43,25 +43,27 @@ To check this control, the following resources may be useful:
 
 ## Example case
 
-We will use Wireshark with [BTVS](https://learn.microsoft.com/en-us/windows-hardware/drivers/bluetooth/testing-btp-tools-btvs) (btvs.exe -Mode wireshark) to capture and analyze the packets.
+In this example, a pairing test with wireless headphones is demonstrated.
 
-The Bluetooth device configuration tool of the laptop's operating system is used to initiate the connection. In Bluetooth Classic, the connection between the two devices starts with the _IO Capability Request_ command.
+For the test, Wireshark has been used with [BTVS](https://learn.microsoft.com/en-us/windows-hardware/drivers/bluetooth/testing-btp-tools-btvs) (btvs.exe -Mode wireshark) to capture and analyze packets. To perform the connection and pairing, the Bluetooth device configuration tool of the laptop's operating system is used.
```

Comment:
Please do not link the tool directly. It would be preferable to link the correct resource (https://www.tarlogic.com/bsam/resources/capture-bluetooth-connection/) as other users may prefer to use other tools. It would be nice to exemplify that BTVS was used but other tools can be used to perform this same technique as documented in https://www.tarlogic.com/bsam/resources/capture-bluetooth-connection/.